src/briefcase/images/images.py

```diff
@@ -0,0 +1,5 @@
+
+def create_variant(target, size, sources):
```

Comment:
Thinking about how this will be used - this function has no context around what a "variant" is. Using a name like `create_image_variant()` would mean that no matter where it is imported, it will be clear what it is for. An "image-less" name would only make sense if you're anticipating having multiple entry points into the `images` module, so that the usage pattern would be `images.create_variant()`; that seems unlikely in this case.